js_machine_front/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "js_machine_front",
-  "version": "0.1.0",
+  "version": "0.2.1",
   "private": true,
   "dependencies": {
     "@types/jest": "24.0.15",
```

Comment:
please relocate @types dependencies to devDependencies, react-scripts and typescript too

Reply:
fix